lib/configuration/configuration.js

```diff
@@ -102,6 +144,13 @@ class Configuration {
   }
 }
 
+const isEventInContext = (eventName, events) => {
+  let eventObject = eventName.split('.')[0]
+  return ((events.indexOf(eventName.trim()) > -1) ||
+    (events.indexOf('pull_request.*') > -1 && eventObject === 'pull_request') ||
+    (events.indexOf('issues.*') > -1 && eventObject === 'issues'))
```

Comment:
missing `schedule.*`

Reply:
i generalized by using `${eventObject}.* instead, this should cover all cases